### PR TITLE
Update deprecate-old-azure-artifacts.md

### PR DESCRIPTION
**Pull request by jorfde**
Call out warnings and upcoming build break dates

### CODE REVIEW

**Comment by Jak-MS** [comment]
@gloridelmorales 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by steved0x** [comment]
#sign-off

File: release-notes/roadmap/deprecate-old-azure-artifacts.md [hunk 1]
```diff
@@ -21,4 +21,10 @@ We now have the following lightweight, auth-only tasks in GA that we recommend c
 1. Maven Authenticate 
 2. NuGet Authenticate
 3. Python Pip Authenticate
-4. Python Twine Upload Authenticate
\ No newline at end of file
+4. Python Twine Upload Authenticate
+
+# Important Migration Notice:
```
Comment (Jak-MS) [inline release-notes/roadmap/deprecate-old-azure-artifacts.md:26]:
```suggestion
## Important Migration notice
```

**Comment by Jak-MS** [comment]
merged after editing

**Pull request merged**